CHANGES.rst

```diff
@@ -321,6 +321,8 @@ Changelog
   [gbastien]
 - Added utils.escape utility function # PM-3462 .
   [odelaere]
+- Disabled active filter on orgs-searches # PM-3228.
```

Comment:
Changer en "Hide the `review_state` filter on orgs-searches contacts dashboard" ?